manifests/a/ASRock/RestartToUEFI/1.0.12/ASRock.RestartToUEFI.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ASRock.RestartToUEFI
+PackageVersion: 1.0.12
+PackageLocale: en-US
+Publisher: ASRock

```

```suggestion
Publisher: ASRock Inc.
```
This should match Control Panel